vyxal/transpile.py

```diff
@@ -36,6 +36,12 @@ def lambda_wrap(
 
 
 def transpile(program: str, dict_compress: bool = True) -> str:
+    """Transpile an entire Vyxal program
+    Parameters:
+    program: str
```

Comment:
Uh, you can kinda tell what parameters it takes by *reading the function header*

Reply:
This much is kinda unnecessary

Reply:
Sure, I'll get rid of this too